tests/Migration/Services/RunServiceTest.php

```diff
@@ -136,6 +141,154 @@ protected function setUp(): void
         ));
     }
 
+    public function testUpdateConnectionCredentialsWhenMigrationIsRunning(): void
+    {
+        $run = new SwagMigrationRunEntity();
+        $run->setId(Uuid::randomHex());
+
+        $this->runRepo = new StaticEntityRepository([
+            new SwagMigrationRunCollection([$run]),
+        ], new SwagMigrationRunDefinition());
+
+        static::expectException(MigrationException::class);
```

Comment:
Sometimes I had read to better use this method `expectExceptionObject()`:

```suggestion
        static::expectExceptionObject(MigrationException::migrationIsAlreadyRunning());
```